### DISCUSSION TITLE
Non bridge XLA open issues

### DISCUSSION THREAD

**Discussion by bhack** [openxla/xla#4] [General]
Do we plan to re-triage and migrate non bridge open XLA issues in this repository?

https://github.com/tensorflow/tensorflow/labels/comp%3Axla

**Comment by hawkinsp** [comment]
It seems like a good idea to do this but it will take time to get there.

**Reply by bhack** [reply]
It will be important also to correctly route end users new tickets between bridges and OpenXLA repos as there could be a tendency to open tickets to the fronted/bridge repo in the end2end experience. 
This will help also to better handling at bridge level a contribution welcome label for bridge only issues. 

/cc @cheshire

**Comment by bhack** [comment]
Do you like to have new tickets like this in TF or in this repo?
https://github.com/tensorflow/tensorflow/issues/58051
